src/components/Empty/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {  HTMLAttributes } from 'react';
+
+export interface EmptyProps extends HTMLAttributes<HTMLSpanElement> {
+  description: string;
+  image: string;
+  children: React.ReactNode;
+}
+
+export const Empty: React.FC<EmptyProps> = ({
+  description,
+  image,
+  children
+}) => {
+  return (
+    <div className='justify-items-center grid grid-cols-1'>
+        <img src={image} alt='EmptyImage'/>
+        <div className='font-mono text-sm text-center'>
```

Comment:
try not to use the mono font, use font style that conforms with the rest of the components in this library